package.json

```diff
@@ -21,7 +21,7 @@
     "clean": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
     "build": "tsc --build",
     "localenv:compose:psql": "docker compose -f ./localenv/cloud-nine-wallet/docker-compose.yml -f ./localenv/happy-life-bank/docker-compose.yml ${BEFORE_MERGED:-} -f ./localenv/merged/docker-compose.yml",
-    "localenv:compose": "BEFORE_MERGED='-f ./localenv/tigerbeetle/docker-compose.yml' pnpm localenv:compose:psql",
+    "localenv:compose": "BEFORE_MERGED='-f ./localenv/tigerbeetle/docker-compose.yml --env-file ./localenv/tigerbeetle/.env.tigerbeetle' pnpm localenv:compose:psql",
```

Comment:
Running `localenv:compose` never actually made tigerbeetle enabled in the backend, since the environment variables in `./localenv/tigerbeetle/.env.tigerbeetle` were never propagated to the `backend` services